app-text/cherrytree/cherrytree-0.39.2.ebuild

```diff
@@ -5,7 +5,7 @@ EAPI=7
 
 PYTHON_COMPAT=( python3_{6,7,8} )
 
-inherit autotools desktop python-any-r1 xdg-utils
+inherit autotools desktop python-r1 xdg-utils
```

Comment:
This doesn't install any python files, so why should it have 
` PYTHON_TARGETS="python3_6 python3_7 python3_8"`

I think `python-any-r1` might suite you better? 
https://dev.gentoo.org/~mgorny/python-guide/any.html